NewSpring/ApollosProject
# Add Saved Prayer List

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds the saved prayer list.

### How do I test this PR?
1. Save a prayer
2. See it in the list
3. Remove it from the list
4. It should be gone.

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
